packages/react-mutation-mapper/src/util/OncoKbUtils.ts

```diff
@@ -345,6 +345,19 @@ export function getTumorTypeName(tumorType?: TumorType) {
     }
 }
 
+export function getTumorTypeNameWithExclusionInfo(
+    tumorType?: TumorType,
+    excludedTumorTypes?: TumorType[]
+) {
+    let name = getTumorTypeName(tumorType);
+    if ((excludedTumorTypes || []).length > 0) {
```

Comment:
`_.isEmpty` might be easier to read here, not a big deal though.

```suggestion
    if (!_.isEmpty(excludedTumorTypes)) {
```